packages/kbn-openapi-bundler/src/openapi_bundler.ts

```diff
@@ -75,11 +80,16 @@ function logSchemas(schemaFilePaths: string[]): void {
   }
 }
 
-async function resolveDocuments(schemaFilePaths: string[]): Promise<BundledDocument[]> {
+async function resolveDocuments(
+  schemaFilePaths: string[],
+  options?: BundleOptions
+): Promise<BundledDocument[]> {
   const resolvedDocuments = await Promise.all(
     schemaFilePaths.map(async (schemaFilePath) => {
       try {
-        const resolvedDocument = await bundleDocument(schemaFilePath);
+        const resolvedDocument = await bundleDocument(schemaFilePath, {
+          includeXLabels: options?.includeLabels,
```

Comment:
Should we have the same name for consistency?

```suggestion
          includeLabels: options?.includeLabels,
```

Reply:
Renamed to `includeXLabels` for transparency.